elastalert/config.py

```diff
@@ -110,11 +116,29 @@ def load_options(rule, conf=None):
     rule.setdefault('query_delay', datetime.timedelta(seconds=0))
     rule.setdefault('timestamp_field', '@timestamp')
     rule.setdefault('filter', [])
+    rule.setdefault('timestamp_type', 'iso')
+    rule.setdefault('_source_enabled', True)
     rule.setdefault('use_local_time', True)
 
+    # Set timestamp_type conversion function, used when generating queries and processing hits
+    rule['timestamp_type'] = rule['timestamp_type'].strip().lower()
+    if rule['timestamp_type'] == 'iso':
```

Comment:
Consider moving these strings into constants, instead of repeating them throughout the codebase.